src/PlantUmlClassDiagramGenerator.Library/RelationshipCollection.cs

```diff
@@ -10,6 +11,14 @@ public class RelationshipCollection : IEnumerable<Relationship>
 {
     private readonly IList<Relationship> items = new List<Relationship>();
 
+    public void AddAll(RelationshipCollection collection)
+    {
+        foreach (var c in collection)
+        {
+            items.Add(c);
+        }
+    }
+
```

Comment:
for addPackageTags we need to save all relations from every file into common list of relations